### PR TITLE
Bump/2.1.3

### PR DESCRIPTION
**Pull request by bollwyvl**
This is a synthesis of the `meta.yaml` changes from #5 for 2.1.3 #8. Do we want to do the intermediate releases? I'd be happy to do that as well, perhaps with a named branch, but would like to get the most up-to-date version up as quickly as possible.

Tests are still not supported: will take a look at packaging `snapshottest`.

### CODE REVIEW

**Comment by carlodri** [comment]
> Tests are still not supported

to me it looks like `snapshottest` is only required to run the `graphene` internal test suite (not to build tests for `graphene`-based apps): are you sure you need to include this in the final package?

I'm not a user of this package, so I might be missing something important here...

**Comment by carlodri** [comment]
Oh, AFAIK you can skip the intermediate versions... In case some user requests them, we can build them in a separate branch.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,53 +1,42 @@
...
   run:
+    - aniso8601 >=3,<4
+    - graphql-core >=2.1,<3
+    - graphql-relay >=0.4.5,<1
     - python
-    - six >=1.10.0
-    - graphql-core >=1.1
-    - graphql-relay >=0.4.5
-    - promise >=2.0
+    - six >=1.10.0,<2
```
Comment (carlodri) [inline recipe/meta.yaml:27]:
```suggestion
    - six >=1.10.0,<2
    - promise >=2.1,<3
```

missing dependency (https://github.com/graphql-python/graphene/blob/d28dc68abca5597f44274e1dbd6d000ff3826084/setup.py#L87)

**Comment by bollwyvl** [comment]
Was just updating with setup.py: The upstream dependency on core, which is
also now promise-ful, should cover this.

**Comment by bollwyvl** [comment]
Also snapshottest is indeed only for the internal tests, but as this is
noarch, adding the test suite helps us catch _something_ and find
dependency version bound changes that simple importing on Linux might
not...

>

**Comment by carlodri** [comment]
> Was just updating with setup.py: The upstream dependency on core, which is
also now promise-ful, should cover this.

I have the feeling that you should still indicate the `promise` dependency, even if some upstream dependency already covers it. But honestly I have no solid argument for this, other than consistency with the package's `setup.py`...

**Comment by bollwyvl** [comment]
Ah, right you are, was looking at master. 2.1.3 is different. Will fix this
evening, or feel free to fix!

**Comment by carlodri** [comment]
you can just accept my suggested change above (it's a cool new GitHub feature)

**Comment by bollwyvl** [comment]
Yeah, alas i don't do real gh stuff in my phone... Will be a few hours
before at a machine with creds...

File: recipe/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipe/meta.yaml:27]: added in sorted order...

**Pull request merged**